converter_subscribers:
  darknet:
    topic: "darknet_ros/bounding_boxes"
    queue_size: 1

converter_publishers:
  output:
    topic: "norfair/converter"
    queue_size: 1
